tests/unit/utils/test_sparams.py

```
@@ -0,0 +1,93 @@
+"""Unit tests for fdtdx.utils.sparams – pure-logic, no JAX."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)